src/protocols/bdx/AsyncTransferFacilitator.h

```diff
@@ -72,7 +72,11 @@ class AsyncTransferFacilitator : public Messaging::ExchangeDelegate
      */
     virtual void DestroySelf() = 0;
 
-    // Calling ProcessOutputEvents can destroy this object before the call returns.
+    /**
+     * Calling ProcessOutputEvents can destroy this object before the call returns for certain cases
+     * where the state machine either receives a message indicating termination of the transfer session
+     * or some error cases where the state machine goes to a bad, unrecoverable state.
```

Comment:
Why this change?  Calling this can just destroy the object, period.  Why it does it is not something API consumers should care about.

Reply:
i thought you said you wanted me to record the error cases here but i think you meant in the API itself. removed.